x/epochs/types/genesis.go

```diff
@@ -13,6 +13,15 @@ func NewGenesisState(epochs []EpochInfo) *GenesisState {
 // DefaultGenesisState returns the default epochs genesis state
 func DefaultGenesisState() *GenesisState {
 	epochs := []EpochInfo{
+		{
+			Identifier:              YearEpochID,
```

Comment:
@loredanacirstea why do you want to introduce yearly epochs? Considering leap years, the epochs wouldn't be accurate using a Duration of `time.Hour * 24 * 365`

Reply:
I added `yearly` because it is frequent in business contracts. If you open this API to smart contracts (like I showed in the cron precompile demos), you can transcribe the business contract into a smart contract.
If the epochs module cannot have yearly, or monthly (also frequently used), maybe we should think about how it can be rewritten, so it can.